config/gulp/config.json

```diff
@@ -44,6 +44,8 @@
             "source/js/production/modules/handleFocus.js",
             "source/js/production/modules/setupTruncateLines.js",
             "source/js/production/modules/altinnLoader.js",
+            "source/js/production/modules/bootstrap-datepicker.nn.js",
+            "source/js/production/modules/bootstrap-datepicker.no.js",
```

Comment:
For English, i assume bootstrap has a default setting?

Reply:
Yes, datepicker has an English default. The "Bokmål" had a typo, and the "Nynorsk" was missing.